src/lib/vm/vm.js

```
@@ -2327,6 +2342,19 @@ export default class VM {
     this.state = {
       ...GlobalInjected,
       ...this.globalFunctions,
+      window: {
+        location: {
+          get hash() {
+            return window.location.hash.substring(1) // skip the first char (#),
+          },
+          set hash(value) {
+            if (value !== "") {
+              return console.error("Set `window.location.hash` only accepts an empty string as a value.");
```

Maybe better to throw the error instead of just returning an console.error